doc/xmlrpc_client.py

```diff
@@ -0,0 +1,21 @@
+import xmlrpc.client
+
+HOST = "localhost"
+PORT = "80"
```

Comment:
Sure this is your port?

Reply:
80 is the local web service (http), 443 is web secure service (https), 8069 is the Odoo service. I think you meant 8069

Reply:
Here is the configuration of my docker-compose.yml

```bash
version: '3.1'
services:
  db:
    image: postgres:alpine
    container_name: postgres_odoo15
    environment:
      - POSTGRES_DB=postgres
      - POSTGRES_USER=odoo
      - POSTGRES_PASSWORD=odoo
    ports:
      - "5432:5432"
  odoo:
    image: odoo:15
    container_name: odoo15
    depends_on:
      - db
    ports:
      - "80:8069"
    volumes:
      - ./config/odoo.conf:/etc/odoo/odoo.conf
      - ./addons:/mnt/extra-addons
```